src/main/java/seedu/duke/constants/Message.java

```diff
@@ -10,6 +10,7 @@ public enum Message {
             + "\t help - show list of commands\n"
             + "\t add CURRENCY AMOUNT - adds that amount of money into that currency account\n"
             + "\t balance [CURRENCY] - view balances of accounts\n"
+            + "\t\t no CURRENCY - displays all currencies\n"
```

Comment:
It would be better to include all the possible use cases and what it does in the sub table. For example
```suggestion
            + "\t\t balance - displays the balances of all accounts\n"
            + "\t\t balance <CURRENCY> - displays the balance of that specific currency account\n"
```